app/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
         versionCode 72
         versionName '2.4.1'
         minSdkVersion project.minSdkVersion
-        targetSdkVersion project.targetSdkVersion
+        targetSdkVersion 23
```

Comment:
This needed to be hardcoded, as otherwise it says "Access to 'project' exceeds its access rights" and the camera issue isn't fixed. There might be a problem with our Gradle constants.